nncf/config_schema.py

```diff
@@ -475,9 +475,6 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
     "concave": with_attributes(_BOOLEAN, description="For polynomial scheduler - if True, then the target sparsity "
                                                      "level will be approached in concave manner, and in convex "
                                                      "manner otherwise."),
-    #"sparsity_init": with_attributes(_NUMBER,
-    #                                 description="Initial value of the sparsity level applied to the "
-    #                                             "model"),
     "sparsity_target": with_attributes(_NUMBER,
                                        description="Target value of the sparsity level for the model"),
     "sparsity_target_epoch": with_attributes(_NUMBER,
```

Comment:
Reformulate description in terms of epoch indexes, please. (and for `sparsity_freeze_epoch` too)

Reply:
Done.